lib/src/component/prop_mixins.dart

```diff
@@ -72,19 +72,26 @@ abstract class _$DomPropsMixin {
 
   int cols, rows, size, span, start;
 
-  bool allowFullScreen, async, autoPlay, checked, controls, defer, disabled, formNoValidate, hidden, loop, multiple,
-    muted, noValidate, open, readOnly, required, seamless, selected;
+  @Accessor(key: 'default')
+  bool htmlDefault;
+
+  @Accessor(key: 'is')
+  String htmlIs;
+
+  bool allowFullScreen, async, autoPlay, checked, controls, defer, disabled, formNoValidate, headers, hidden,
```

Comment:
In this group there's headers, inputMode, integrity, keyParams, keyType, and kind which were labeled as `String` in our spreadsheet / React. Was there a reason that they ended up here?

Reply:
Yeah I don't know why I put them there haha. They definitely should be strings. Thank you for catching that!